src/index.html

```diff
@@ -9,11 +9,328 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <header class="header">
+    <a href="#" class="logo-link">
+      <img src="./images/logo.png" class="navbar-brand" alt="">
```

Comment:
Attr `alt` can't be empty

Reply:
Description in tag "alt" is added: 
![image](https://user-images.githubusercontent.com/32244670/62511734-b747a900-b81d-11e9-9ad3-dd98b70699f8.png)